### DISCUSSION TITLE
Level Up Together!

### DISCUSSION THREAD

**Discussion by doonfrs** [doonfrs/pluto_grid_plus#56] [Ideas]
Hi,

We initially focused on keeping the package upgraded and usable. However, we've also managed to add some great features! This success has me considering taking things to the next level.

Here are some ideas for improvement:

- [ ] Create a dedicated Telegram channel for package maintainers to facilitate ongoing conversation.
- [ ] Host a new public demo to showcase the web version's capabilities.
- [ ] Enhance the existing code documentation for improved clarity.
- [ ] Create a new, up-to-date developer manual reflecting the latest features.

Please share your thoughts and ideas for further enriching this project 🙏🙏🙏

**Comment by stan-at-work** [comment]
That's a great idea. But i think the main focus needs to be to fix the backlog of issues. If a package is not fixing its problem's there is no need to add extra features,

**Comment by VittorioParagallo** [comment]
I think it's a great idea, but we should bring people commiting on pluto_grid here.
